### ISSUE TITLE
There is a directory /.dotnet

### ISSUE THREAD

**Issue by ststeiger** [dotnet/sdk#12544]
### Description

On my Linux system, there is a directory /.dotnet.  
Since it contains optimizationdata, I presume it is created by runtime. 

This doesn't belong here. 
This should probably be in /var/lib/dotnet or /usr/lib/dotnet or /usr/share/.dotnet

**Comment by ststeiger** [comment]
> Did you happen to run dotnet SDK commands in environment without home directory or with home directory set up to point to /? The container environments are sometimes setup that way.

No, I run it on a kUbuntu 18.04 system, without docker or container. 
This is my dev system. 
Home is set to /root.

**Comment by ststeiger** [comment]
I have recently tried 
https://github.com/microsoft/ClangSharp
and 
https://github.com/mono/CppSharp
which has not worked. 
It might be from there, ClangSharp failed to build anything.

**Comment by marcpopMSFT** [comment]
Please let us know if this is still an issue with newer sdks (7.0.1xx)
